src/components/security/VulnerabilityExposure.tsx

```diff
@@ -14,473 +14,319 @@
+                    title="CVE Policy"
+                    renderDescriptionContent={() =>
+                        'Check the exposure of your system for a CVE-ID for future deployments.'
+                    }
+                    showInfoIconTippy
+                    dataTestId="cve-exposure-heading"
+                />
+            </div>
+            <div className="px-20 py-16 flexbox-col dc__gap-16">
+                <div className="flexbox dc__gap-8">
+                    <SearchBar
+                        initialSearchText={cveName}
+                        containerClassName="w-250 br-4"
+                        handleSearchChange={setCveNameSearchKey}
+                        handleEnter={handleSearchCVE}
+                        inputProps={{
+                            placeholder: 'Enter CVE ID',
+                            autoFocus: true,
+                        }}
+                        dataTestId="security-vulnerability-cve-list"
+                    />
+                    <button className="cta small" type="button" onClick={handleSearchCVEButtonClick}>
+                        Search
+                    </button>
                 </div>
-                <form className="vulnerability-exp__form  ml-24 mr-16" onSubmit={this.searchCVE}>
-                    <div className="flexbox dc__gap-8">
-                        <SearchBar
-                            initialSearchText={this.state.form.cve}
-                            containerClassName="w-250 br-4"
-                            handleSearchChange={this.handleCVEChange}
-                            handleEnter={this.handleCVESearchFilterKeyUp}
-                            inputProps={{
-                                placeholder: 'Enter CVE ID',
-                                autoFocus: true,
-                            }}
-                            dataTestId="security-vulnerability-cve-list"
-                        />
-                        <button className="cta small">Search</button>
-                    </div>
-                    {this.state.cve ? (
-                        <p className="mt-16 mr-24 mb-0">
-                            Showing results for&nbsp;
-                            <a
-                                href={`https://cve.mitre.org/cgi-bin/cvename.cgi?name=${this.state.form.cve}`}
-                                rel="noopener noreferrer"
-                                target="_blank"
-                            >
-                                {this.state.cve}
-                            </a>
-                        </p>
-                    ) : null}
-                </form>
-                <hr className="mt-0 mb-0" />
-                {this.renderList()}
+                {!!cveName && (
+                    <span className="fs-13">
+                        Showing results for&nbsp;
+                        <a
+                            href={`https://cve.mitre.org/cgi-bin/cvename.cgi?name=${cveName}`}
```

Comment:
Can we create a common util to get this Link have seen this duplicated at many places, we can replace it from there as well